.github/workflows/security-fuzz.yml

```
@@ -0,0 +1,97 @@
+name: Security Fuzz
+
+on:
+  push:
+    branches: [ "**" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)